legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-test/src/test/java/org/finos/legend/engine/persistence/components/BaseTest.java

```diff
@@ -339,17 +343,68 @@
         .addFields(validityThroughTarget)
         .build();
 
+    protected SchemaDefinition bitemporalMainTableSchemaWithBatchIdAndTime = SchemaDefinition.builder()
+            .addFields(id)
+            .addFields(name)
+            .addFields(amount)
+            .addFields(validityFromReference)
+            .addFields(validityThroughReference)
```

Comment:
Under our new setting, the main table should not have validityFromReference and validityThroughReference anymore

Reply:
Agreed, removed these two fields!